frontend/src/app/main/ui/workspace/sidebar/options/menus/stroke.cljs

```diff
@@ -76,7 +76,7 @@
          (mf/deps ids)
          (fn [index color]
            (let [color (-> color
-                           (assoc :id nil :file-id nil))]
+                           (dissoc color :ref-id :ref-file))]
```

Comment:
I think color is duplicated, I suggest remove the `(-> color ..`

Reply:
Thanks!